README.md

```diff
@@ -25,6 +25,39 @@ See this [issue](https://github.com/react-native-google-signin/google-signin/iss
 
 Then follow the [Android guide](docs/android-guide.md) and [iOS guide](docs/ios-guide.md)
 
+## Expo installation
+
+> This package cannot be used in the "Expo Go" app because [it requires custom native code](https://docs.expo.io/workflow/customizing/).
```

Comment:
this sounds a little like "you can't use this in expo" while then proceeding with expo installation instructions which is a little confusing.

Can you maybe add a sentence like "This package cannot be used in the "Expo Go" app because it requires custom native code. _However, you can add custom native code to expo by following ..."